server/trucks.js

```diff
@@ -253,4 +253,15 @@ var foodTrucks = [
  * that method should take a single parameter that represents the day to filter on
  * use underscore's filter() method to return all trucks that have the day in their
  * schedule
- * /
\ No newline at end of file
+ */
+var undsc = require('../node_modules/underscore/underscore');
+
+var filterByDay = function (day) {
+    var trucks = undsc.filter(foodTrucks, function (day) {
+        foodTrucks.schedule == day;
+        console.log(foodTrucks.schedule);
+    });
+    return trucks;
+};
+
+module.exports.filterByDay = filterByDay();
```

Comment:
Don't invoke filterByDay here, just assign a reference to it so it can be called elsewhere.